compiler/resolution/functionResolution.cpp

```diff
@@ -1628,6 +1628,166 @@ computeGenericSubs(SymbolMap &subs,
 }
 
 
+//
+// Is 'parent' an indexing expression into the tuple 'se'?
+// If so, return the index (between 1 and numArgs). Otherwise return 0.
+//
```

Comment:
1) There's inconsistent formatting here with respect to use of spaces before "{" and "(".

2) There an inconsistency about where the "{" appears (same line in most cases, next line in one case)

3) As a good rule of thumb, which is easily applied here, the components of the conditional for the third if
statement should either be on one line or on individual lines.

4) I think this name of this function is confusing. The pattern is<FOO> would suggest it's a predicate but then
it returns an integer. It's clear there are pros/cons to overloading the return value to mean both pass/fail and
index if pass.  If pursuing the overloading path then bias things for the expected case i.e. pass.  This is especially
easy to defend for a static function in which the caller and callee are under tight control.